kolibri/plugins/pwa/kolibri_plugin.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2023 Endless OS Foundation, LLC
```

Comment:
We don't have an explicit contributor license grant in place, but we can't accept code into our code base that is under copyright to another organization, without an associated license disclaimer.

My preference would be for this copyright notice to be removed.

Reply:
As discussed on Slack, I’ve kept the copyright line and also added a one-line license identifier (in all new files).